kie-server-parent/kie-server-controller/kie-server-controller-test-war/src/main/assembly/assembly-ee6-container.xml

```diff
@@ -40,9 +40,6 @@
 
   <dependencySets>
     <dependencySet>
-      <excludes>
-        <exclude>org.jboss.resteasy:*</exclude>
-      </excludes>
```

Comment:
This exclusion (and the other one) were in fact not excluding anything. assembly plugin was complaining about that.